app/javascript/components/responseTypes/TextResponse.js

```diff
@@ -0,0 +1,31 @@
+
+import * as React from 'react';
+import TextField from '@mui/material/TextField';
+import { Typography } from '@mui/material';
+
+class TextResponse extends React.Component {
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
```

Comment:
Just to make sure this is how we should be creating our components? The component only returns the required front end stuff it does not hold any data?

Reply:
I only ask because it makes the McQuestion a bit more awkward to make since it cannot internally store the list of options

Reply:
In this case your component here only returns the MUI stuff required for the front end.

Reply:
oh nevermind I see it stores the question